scripts/HPC_chatbot_preprocessor/parsed_mds/generic/apptainer/apptainer_paragraph_1.txt

```diff
@@ -0,0 +1,42 @@
+Apptainer (formally known as Singularity)
+What is Apptainer?
+Apptainer is an open-source computer program that performs
+operating-system-level virtualization (also known as containerisation).
+One of the main uses of Apptainer is to bring containers and
+reproducibility to scientific computing and the high-performance
+computing (HPC) world. Using Apptainer/Singularity containers,
```

Comment:
remove the `/Singularity`. maybe add a small section or tip that it was used to be called singularity, and that you might still find refenrences to singularity